### ISSUE TITLE
Improve documentation on value type in Python API

### ISSUE THREAD

**Issue by erikbosch** [eclipse/kuksa.val#493]
As of today most methods in the Python API does not provide any type hints, and does not provide any comments either. Taking the example below it is difficult to deduce what types are allowed for "value".  At least if using websocket/VISS API, the value must be a JSOn object: str, bytes or bytearray. Trying to use a Python int or bool will give an error.

I do not know if we as of now expect it to be called with bytes/bytearray as input, or if we always expect a string? If so we could possibly use str as type hint.

https://github.com/eclipse/kuksa.val/blob/master/kuksa-client/kuksa_client/__init__.py#L70

```
    # Set value to a given path
    def setValue(self, path, value, attribute="value", timeout=5):
        return self.backend.setValue(path, value, attribute, timeout)
```

**Comment by lukasmittag** [comment]
```
# Update Meta Data of a given path
    def updateMetaData(self, path, jsonStr, timeout=5):
        return self.backend.updateMetaData(path, jsonStr, timeout)
```
This could be a good example we named this variable ```jsonStr``` but it accepts Any type in function call.

**Comment by lukasmittag** [comment]
solved by #529
